source/UF/Truncations.lagda

```diff
@@ -265,6 +298,29 @@ We define the canonical predecessor map and give a computation rule.
 
 \end{code}
 
+We will show that any type X is equivalent to the sigma over a path space
+with one truncated endpoint and one free endpoint.
+
+\begin{code}
+
+ equiv-trunc-path-space : {X : 𝓤 ̇} {n : ℕ₋₂}
```

Comment:
You usually write `truncated` instead of `trunc`, which I prefer.

Reply:
This is a direct instance of `domain-is-total-fiber` from `UF.EquivalenceExamples`. Is it worth recording separately?

Reply:
Originally,  I thought I would use it in a later proof, but I was mistaken so it did not prove to be worth recording seperately. I will remove it.